src/plugins/data/public/ui/filter_bar/filter_editor/index.tsx

```diff
@@ -486,7 +486,7 @@ class FilterEditorUI extends Component<Props, State> {
         operator.type,
         operator.negate,
         this.props.filter.meta.disabled,
-        params,
+        params || '',
```

Comment:
I think it's valid that the value of `0` could get in here, in which case we'd probably want to switch this to the following (which is probably better anyway):

```suggestion
        params ?? '',
```